docs/guide/image_editor.md

```diff
@@ -87,3 +87,16 @@ new FieldValue([
      ],
  ]),
 ```
+
+### Image quality
```

Comment:
The things I'd change is:
Move the image quality setting before the Additional information" section. Image quality is a readily available setting thet you do in the same place where you configure actions, while "Additional information" is an extension point, not directly related to how the editor works.

Also - what does quality mean? Is it compression rate of a JPEG image? Could we explain this? If so, does 1 correspond to 100% compression rate? Could we then say "...where .92 corresponds to 92% compression rate"?

Not overly attached to this, though.

Reply:
Makes sense with the order of info, will move.
As for quality, I hope @ViniTou can elaborate?

Reply:
I just created setting on our side, those questions should be addressed to @dew326.

But i would say its other way around, `0` is the strongest compression.

Reply:
>A Number between 0 and 1 indicating the image quality to use for image formats that use lossy compression such as image/jpeg and image/webp.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL - encoderOptions